libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -7,9 +7,15 @@
 #include <AP_Baro/AP_Baro.h>
 
 class AP_Airspeed_Backend;
+class AP_AHRS;
 
 #define AIRSPEED_MAX_SENSORS 2
 
+ // set the limit to 40% (TBD)
```

Comment:
Why is this still TBD? Has it not been concluded either way at this point? I can only think it hasn't because of the lack of real world flight data.

Reply:
Exactly. If you have logs with pitot failure, bring them in. I had a crash already due to icing and difference was 80%.